src/utils/AppRouter.js

```diff
@@ -12,17 +12,24 @@ import JobDetails from '../job-details/JobDetails';
 import UsersTable from '../admin-page/UsersTable';
 import { AuthContext } from '../context/AuthProvider';
 import userRoleMapping from './constants';
+import MyProfilePage from '../myprofile-page/MyProfilePage';
 
 const AppRouter = () => {
   const { userRole } = useContext(AuthContext);
 
-  const isRouteAvailable = (requiredRole) => requiredRole === userRoleMapping[userRole];
+  const isRouteAvailable = (requiredRole) => {
+    if (requiredRole === 'any') {
```

Comment:
you may add this to the userRoleMapping instead of having an if.

Reply:
updated